cvat/apps/dataset_manager/formats/mot.py

```diff
@@ -45,6 +45,17 @@ def _import(src_file, task_data):
 
                 track_id = ann.attributes.get('track_id')
                 if track_id is None:
+                    # Extension. Import regular boxes:
+                    task_data.add_shape(task_data.LabeledShape(
```

Comment:
Why don't create a track with only one bounding box?

Reply:
Is it any better? This would not preserve the original annotations, as all tracks are exported as tracks, even single-framed. We can, possibly, convert regular boxes to such tracks.

Reply:
MOT is for tracks only. Right? How are you going to dump a track which has two shapes:
- First with outside=False, frame=0
- Second with outside=True, frame=1?

Reply:
MOT is mostly for tracks, look here: https://motchallenge.net/instructions/. Almost every version has `<[track] id>` field set (idx 2), but not `MOT17det`, which contains just boxes and `-1` as `id`. So we can safely assume that bboxes are supported via this extension.

Reply:
OK